README.md

```diff
@@ -9,7 +9,7 @@
 
 An Aerospike add-on module for Node.js.
 
-This module is compatible with Node.js v4.x (LTS), v6.x (LTS) and v7.x. It
+This module is compatible with Node.js v4.x (LTS), v6.x (LTS) and v8.x (LTS). It
```

Comment:
According to the [LTS schedule](https://github.com/nodejs/LTS), Node.js v8 doesn't actually become an active LTS version until October 2017.